packages/block-editor/src/components/text-transform-control/index.js

```diff
@@ -42,26 +45,31 @@ const TEXT_TRANSFORMS = [
  *
  * @return {WPElement} Text transform control.
  */
-export default function TextTransformControl( { value, onChange, ...props } ) {
+export default function TextTransformControl( { value, onChange } ) {
 	return (
-		<ToggleGroupControl
-			{ ...props }
-			className="block-editor-text-transform-control"
-			__experimentalIsBorderless
-			label={ __( 'Letter case' ) }
-			value={ value }
-			onChange={ onChange }
-		>
-			{ TEXT_TRANSFORMS.map( ( textTransform ) => {
-				return (
-					<ToggleGroupControlOptionIcon
-						key={ textTransform.value }
-						value={ textTransform.value }
-						icon={ textTransform.icon }
-						label={ textTransform.name }
-					/>
-				);
-			} ) }
-		</ToggleGroupControl>
+		<fieldset className="block-editor-text-transform-control">
```

Comment:
It might still be handy to allow passing additional props through here, or at least a custom `className` so we reuse the existing `.edit-site-typography-panel__full-width-control` style to make this control span the full width of the panel as mentioned in the discussion on this PR.

Reply:
I'll add back `className`. We don't need it for that exact purpose though because `single-column` is added to the `ToolsPanelitem` in block settings and, for some reason, a wrapping div in global styles.